test/testdata/infer/generics/disabled/variance_params.rb

```diff
@@ -0,0 +1,50 @@
+# typed: true
```

Comment:
why is this test disabled? What part of it fails?

Reply:
It only fails in LSP, as the fast-path doesn't raise the error about non-invariant type members.

Reply:
Can we leave a comment on this?

Reply:
(or make classes into modules)

Reply:
That worked, thanks!